store-implementation/federated-store/src/main/java/uk/gov/gchq/gaffer/federatedstore/operation/handler/FederatedAddGraphHandlerParent.java

```diff
@@ -98,10 +100,11 @@ private void overwriteCacheProperty(final OP operation, final Store store) {
          * FederatedStore can't survive if a subgraph changes the static
          * cache to another cache, or re-initialises the cache.
          */
+        final String cacheServiceClass = isNull(store.getProperties()) ? null : store.getProperties().getCacheServiceClass(CACHE_SERVICE_CLASS_DEFAULT);
         final StoreProperties storeProperties = operation.getStoreProperties();
```

Comment:
One line uses `store.getProperties()`, the other `operation.getStoreProperties()`. Isn't this the same thing? Better to be consistent.

Reply:
no one is FederatedStores Property the other is the Property of the store being added.

I want overwrite with one.